src/encoded/static/components/static-pages/components/TableOfContents.js

```diff
@@ -3,20 +3,20 @@
 import React from 'react';
 import * as d3 from 'd3';
 import _ from 'underscore';
+import memoize from 'memoize-one';
 import { Collapse, Button } from 'react-bootstrap';
-import * as globals from './../../globals';
 import { getElementTop, animateScrollTo, getScrollingOuterElement, getPageVerticalScrollPosition } from './../../util/layout';
 import { isServerSide, console, navigate, object } from './../../util';
 
 
-class TableEntry extends React.Component {
+class TableEntry extends React.PureComponent {
 
-    static getChildHeaders(content, maxHeaderDepth, currentDepth){
+    static getChildHeaders = memoize(function(content, maxHeaderDepth, currentDepth){
```

Comment:
On 2nd thought, this memoization isn't really needed here because static fxns are getting called by multiple different table of content entries so is never really memoized.

Memoization could be made working/effective maybe by hosting an instance of a memoized func in each table of contents entry component instance but is more refactoring that doesn't add much benefit/performance so prly not worth doing.